examples/pytorch/graphsaint/README.md

```diff
@@ -0,0 +1,118 @@
+# GraphSAINT
+
+This DGL example implements the sampling method proposed in the paper [GraphSAINT: Graph Sampling Based Inductive Learning Method](https://arxiv.org/abs/1907.04931).
+
+Paper link: https://arxiv.org/abs/1907.04931
+
+Author's code: https://github.com/GraphSAINT/GraphSAINT
+
+Contributor: Liu Tang ([@lt610](https://github.com/lt610))
+
+## Dependecies
+
+- Python 3.7.0
+- PyTorch 1.6.0
+- numpy 1.19.2
+- dgl 0.5.3
+
+## Dataset
+
+All datasets used are provided by Author's [code](https://github.com/GraphSAINT/GraphSAINT). They are available on [Google Drive link](https://drive.google.com/drive/folders/1zycmmDES39zVlbVCYs88JTJ1Wm5FbfLz) (alternatively, [BaiduYun link (code: f1ao)](https://pan.baidu.com/s/1SOb0SiSAXavwAcNqkttwcg#list/path=%2F)). Dataset summary("m" stands for multi-class classifification, and "s" for single-class.):
+| Dataset | Nodes | Edges | Degree | Feature | Classes | Train/Val/Test |
+| :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| PPI | 14,755 | 225,270 | 15 | 50 | 121(m) | 0.66/0.12/0.22 |
+| Flickr | 89250 | 899756 | 10 | 500 | 7(s) | 0.50/0.25/0.25 |
+| Reddit | 232965 | 11606919 | 50 | 602 | 41(s) | 0.66/0.10/0.24 |
+| Yelp | 716847 | 6877410 | 10 | 300 | 100(m) | 0.75/0.10/0.15 |
+| Amazon | 1598960 | 132169734 | 83 | 200 | 107(m) | 0.85/0.05/0.10 |
+
+## Full graph training
+
+Run with following:
+```bash
+python train_full.py --gpu 0 --dataset ppi --n-epochs 1000 --n-hidden 512 --arch 1-0-1-0 --batch-norm
+python train_full.py --gpu 0 --dataset flickr --n-epochs 100 --n-hidden 256 --arch 1-1-0 --batch-norm --dropout 0.2
+python train_full.py --gpu -1 --dataset reddit --n-epochs 100 --n-hidden 128 --arch 1-0-1-0 --batch-norm --dropout 0.1
+python train_full.py --gpu -1 --dataset yelp --n-epochs 100 --n-hidden 512 --arch 1-1-0 --batch-norm --dropout 0.1
+python train_full.py --gpu -1 --dataset amazon --n-epochs 100 --n-hidden 512 --arch 1-1-0 --batch-norm --dropout 0.1
```

Comment:
How did you get the hyperparameters here?

Reply:
I used the author's hyperparameters.